dev/breeze/src/airflow_breeze/commands/developer_commands.py

```diff
@@ -546,31 +522,17 @@ def enter_shell(**kwargs) -> RunCommandResult:
     * executes the command to drop the user to Breeze shell
 
     """
-    verbose = kwargs["verbose"]
-    dry_run = kwargs["dry_run"]
-    perform_environment_checks(verbose=verbose)
+    perform_environment_checks()
     if read_from_cache_file("suppress_asciiart") is None:
         get_console().print(ASCIIART, style=ASCIIART_STYLE)
     if read_from_cache_file("suppress_cheatsheet") is None:
         get_console().print(CHEATSHEET, style=CHEATSHEET_STYLE)
-    enter_shell_params = ShellParams(**filter_out_none(**kwargs))
-    rebuild_or_pull_ci_image_if_needed(command_params=enter_shell_params, dry_run=dry_run, verbose=verbose)
-    if enter_shell_params.include_mypy_volume:
+    shell_params = ShellParams(**filter_out_none(**kwargs))
+    rebuild_or_pull_ci_image_if_needed(
+        command_params=shell_params,
+    )
```

Comment:
may as well drop the comma here so this will collapse down to a one-liner

```suggestion
    rebuild_or_pull_ci_image_if_needed(command_params=shell_params)
```